.travis.yml

```diff
@@ -1,25 +1,18 @@
-language: node_js
-node_js:
-    - '10'
-env:
-    global:
-        - CC_TEST_REPORTER_ID=b783cd121c630ba19b3af10ad4d883274eb4e942a0eacf53107ba50ca5f67897
-        - secure: J0hYjgHzqGcXE5IYbkYKWGhQDPGrnWKB8zOuz8jcTivzYHYAz+IB4j6Hg9Misp9SNsTovom8OUn4yv4ublz5zUys3v7YgSmb1sucyRAfrUzMAEdqUmCA/pNRCddpb+mPZ1v5JXfzi/ZTC7GnY9LojKw4Gf+P3ItZhGhjCE5oGMbWIdQ2FjtZVxp48zj0Zq/++ZTUeGfT+jpHGhmdNWPrQDtjHZ/Xh/WNnWXWW4fshv3cxIcOcyFjjxN6Ay3nNLGwecVpejlSM3EGfFe2ho1YUvp2fdta/fVr1tp4wVh6zlBvhTEavgWD4G2TQUbRAvZn0g65mS6UdsqaFNAdQ2965EkcSwViyJ8z9Em9kRkMe/CBH0SvA5MDSf99H8gnlkHH2Id66s6Q876nNHsCvI+WFRaiZvbtCQ57u9w/jSEpZEl7c64Xsk6ZCdrsTTJpqLjotxB8KmdnbaO+SEhtVIK8eUZZzh0pWrFvzO8i+7+oYkXJNEaZ7QBZl3WAT5ZtMl/LWs+ihpF9PpwE5IOkxVWqIgEND4fwfhkoPUqxLMNA6eHO7hyfuTSUDT5HPt4LKl1TUorFyPEwITdITksK0djgMJxkNzfnt0DnETSVODEOFQi1WUDVwq0jckI/OSbWNsqZmLkJ/7mACvQJq5oEI2zywTdH2y/ha/zuBhZ8ddeZ3QQ=
-before_script:
-    - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64
-      > ./cc-test-reporter
-    - chmod +x ./cc-test-reporter
-    - './cc-test-reporter before-build'
-    - npm install --global @dhis2/deploy-build
+language: node_js	
+node_js:	
+    - '10'	
```

Comment:
We could do it like this instead: https://github.com/dhis2/ui-core/blob/master/.travis.yml#L3

Just to avoid having to go through everything when there's a new LTS.

Reply:
Yeah sounds good, I've changed it 👍